index.html

```diff
@@ -1,6 +1,17 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
Why are you making changes to this file? What do these changes do?

Reply:
The link below gave me instructions of how to embed the font into the app.
https://fonts.google.com/selection/embed
That is why I made any changes in the index.html or index.css files